### PR TITLE
DDF-1675 Fix unreliable documentation processing

### PR DESCRIPTION
**Pull request by brendan-hofmann**
@pklinef @AzGoalie

[Review on Reviewable](https://reviewable.io/reviews/codice/ddf/354)

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/codice/ddf/pull/354" target="_blank">ReviewNinja

**Comment by pklinef** [comment]
:+1:

File: distribution/ddf/pom.xml [hunk 1]
```diff
@@ -126,7 +126,16 @@
...
                             <execution>
                                 <id>copy-docs</id>
-                                <phase>prepare-package</phase>
+                                <!--
+                                The unpack needs to run in an earlier phase than the copy,
+                                due to a recurring bug in maven that causes plugins in the
+                                same lifecycle phase to run in orders other than that which
+                                they were declared in.
+
+                                See https://issues.apache.org/jira/browse/MNG-5799 as well
```
Comment (shaundmorris) [inline distribution/ddf/pom.xml]: @brendan-hofmann thanks for adding the descriptive comment in there as to we did it this way

**Comment by ricklarsen** [comment]
:+1:

**Comment by oscaritoro** [comment]
:+1:

**Pull request merged**